src/WebUI/locales/en.yml

```diff
@@ -224,6 +225,22 @@ character:
       title: Play Time
     name:
       title: Level
+    rank:
+      title: Rank
+      tooltip:
+        title: Rank
+        # prettier-ignore
+        desc: "
+          <p>Your MMR assigns you a rank, which is tied directly to the autobalancer, using this it decides which team to place you on. The system we use is <strong>*Glicko2</strong>.</p>
```

Comment:
I would avoid using MMR since it means match making rating and there is no match making. Just "rating" will work.